tox.ini

```diff
@@ -1,43 +1,36 @@
 [tox]
-skip_missing_interpreters=true
-envlist=py36,py37,py38,coverage,docs
+skip_missing_interpreters = true
+envlist =
+    py36,py37,py38,py39,
+    coverage,
+    docs
 
 [testenv]
-skip_install=true
 setenv =
-  LC_ALL=C
-  LANG=C
-  COVERAGE_FILE=.coverage.{envname}
-deps =
-  pytest
-  -e .[tests,docs]
-  lxml
-  pyquery
-  BeautifulSoup4
-  PasteDeploy
-  WSGIProxy2
-  coverage
+    LC_ALL=C
+    LANG=C
+    COVERAGE_FILE=.coverage.{envname}
+extras =
+    tests
 commands =
-  python --version
-  pip freeze
-  coverage run {envbindir}/pytest -xv []
-  coverage report --show-missing
+    python --version
+    pip freeze
+    coverage run {envbindir}/pytest -xv []
```

Comment:
This doesn't work on Windows. Instead I'd recommend running `pytest --cov --cov-report=term-missing` or whatever we do in other Pylons projects. (put flags in `setup.cfg`: https://github.com/Pylons/waitress/blob/master/setup.cfg#L68)

Running coverage stand-alone is not necessary.

Reply:
This will also fix your Windows issues.

Reply:
Thanks @bertjwregeer! I pushed another commit and we'll see how the builds go. 🤞 

I also wondered why coverage is run on each platform, other than it might be nice information. This information has been ignored for over 7 years in this project, so I reckon I should remove them here and keep the separate coverage environment.

Reply:
Oops, looks like pytest-cov plugin needs to be installed.

Reply:
Yay! We got Windows! I just hope it's not contagious.

There's one failing test on the Windows builds, something to do with file uploads in the forms.py. That's fine by me because the point of this PR is to get CI to work.

Reply:
Hmm, it looks like Windows barfs on the new line in the uploaded file, where it compares `\n` against a line break in the rendered HTML. I'm not sure how to fix this one.

Reply:
See my comment on the main PR.